ush/getsystem

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
```

Comment:
In NCEPLIBS we have a documentation standard: every file is documented, with a sentence or two of description, and the name of the programmers who worked on the code.

Do we know who made the changes to this file, so we can get that information in?

Reply:
As it it happens, that was me that created the python version of gestsystem in/around March 2021.

Reply:
@AlexanderRichert-NOAA is there anything you can't do? ;-)

How about a one-sentence description for the documentation?